plugins/carla/pluginlistdialog.cpp

```diff
@@ -0,0 +1,1661 @@
+/*
+ * Carla plugin host, adjusted for OBS
+ * Copyright (C) 2011-2023 Filipe Coelho <falktx@falktx.com>
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#include <CarlaBackendUtils.hpp>
+#include <CarlaString.hpp>
+#include <CarlaUtils.h>
+
+#include <QtCore/QDir>
+#include <QtCore/QFileInfo>
+#include <QtCore/QPointer>
+
+#include "pluginlistdialog.hpp"
+#include "pluginrefreshdialog.hpp"
+
+#include "common.h"
+#include "qtutils.h"
+
+CARLA_BACKEND_USE_NAMESPACE
+
+// ----------------------------------------------------------------------------
+// check if the plugin IO makes sense for OBS
```

Comment:
In general we require even single line comments to use the block comment form `/* [...] */` - applies throughout this file an others.